packages/yew/src/platform/pinned/mpsc.rs

```diff
@@ -114,31 +102,34 @@ impl<T> Stream for UnboundedReceiver<T> {
 
 impl<T> FusedStream for UnboundedReceiver<T> {
     fn is_terminated(&self) -> bool {
-        self.inner.items.is_empty() && self.inner.closed
+        // SAFETY: This function is not used by any other functions and hence uniquely owns the
+        // reference.
+        let inner = unsafe { &*self.inner.get() };
+        inner.items.is_empty() && inner.closed
     }
 }
 
 impl<T> Drop for UnboundedReceiver<T> {
     fn drop(&mut self) {
         // SAFETY: This function is not used by any other functions and hence uniquely owns the
         // mutable reference.
-        let inner = unsafe { &mut *self.inner.get_mut_unchecked() };
+        let inner = unsafe { &mut *self.inner.get() };
```

Comment:
Could use `.get_mut()` here, as `self` is also borrowed mutably, to avoid 1 unsafe.

Reply:
We cannot do this as inner is wrapped in an `Rc`.

Reply:
Oops didn't notice the auto-deref :)